packages/tradershub/src/features/cfd/screens/CFDCompareAccounts/constants.ts

```diff
@@ -34,5 +34,5 @@ export const Jurisdiction = {
 export const platformLabel = {
     CTrader: 'Deriv cTrader',
     MT5: 'MT5 Platform',
-    OtherCFDs: 'Other CFDs Platform',
+    DerivX: 'Deriv X',
```

Comment:
No more calling other cfd platforms

Reply:
@thisyahlen-deriv I've already asked @hirad-deriv to confirm it with their squad designers since it's different between design and prod. If you got any confirmation from anyone on this, please let me know.